WooCommerce/Classes/POS/Presentation/Toolbar/POSToolbarView.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+
+struct POSToolbarView: View {
+    @Environment(\.presentationMode) var presentationMode
```

Comment:
```suggestion
    @Environment(\.presentationMode) private var presentationMode
```

Reply:
Good catch, updated in https://github.com/woocommerce/woocommerce-ios/pull/13000/commits/f0070f254c4ffecbaeab7db72a25c26b875e344f